content/en/security/code_security/troubleshooting/_index.md

```diff
@@ -141,6 +141,13 @@ npm install -g @datadog/datadog-ci
 datadog-ci sbom upload /path/to/sbom-file.json
 ```
 
+### Services or teams in SCA libraries are not updating
+
+Service and team in Software Composition Analysis (SCA) results are based on the `entity.datadog.yml` or `CODEOWNERS` files from your repository’s default branch.
+If you’ve made changes to these files in a feature branch, those updates will not be reflected in the vulnerability or library data for that branch.
+
+After updating either file on your default branch, it may take up to 6 hours for the changes to appear in subsequent scan results.
```

Comment:
```suggestion
After updating either file on your default branch, it may take up to six hours for the changes to appear in subsequent scan results.
```